temporalnexus/operation.go

```diff
@@ -53,6 +53,8 @@ import (
 	"go.temporal.io/sdk/workflow"
 )
 
+var errConflictPolicyUseExistingNotSupported = errors.New("workflow ID conflict policy UseExisting is not supported for Nexus WorkflowRunOperation")
```

Comment:
nit: you can just define this inline, no need for this variable.

Reply:
Isn't there a lint that complains about inline errors? Did we remove that?

Reply:
Yeah no need if it isn't even exported. We have no lint error for inline errors